wrt/wrt-uxmanu-android-tests/wrt-ux-app/testscripts/Crosswalk_WebApp_Integration_Switch.feature

```diff
@@ -0,0 +1,15 @@
+Feature: wrt-ux-app
+ Scenario: Crosswalk WebApp Integration Switch
+  When launch "touch_gesture_click"
+   Then I should see title "World Wide Web Consortium (W3C)" in 60 seconds
+    And launch "wrt-ux-app"
+   Then I should see title "test"
+    And I press "recent" hardware key
+    And I save relative view "className=android.widget.ImageView" on the "right" side of view "text=touch_gesture_click" to object "touch_app"
+    And I click saved object "touch_app"
+    And I wait 5 seconds
+   Then I should see view "description=CollapseW3C Opens New Australia Office"
```

Comment:
A new step to check view in some timeout is required here to reduce execution time. Please draft a request to @jacky-young then improve this test.